virt/managing_vms/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -43,6 +44,28 @@ include::modules/virt-running-ssp-pipeline-web.adoc[leveloffset=+2]
 
 include::modules/virt-running-ssp-pipeline-cli.adoc[leveloffset=+2]
 
+[id="virt-deprecated-tasks_{context}"]
+== Removing deprecated or unused resources
+
+You can clean up Tekton-Tasks-Operator resources associated with the operator.
```

Comment:
```suggestion
You can clean up deprecated or unused resources associated with the {pipelines-title} Operator.
```
A few comments:

- Where possible, use the attributes for product names. See https://redhat-documentation.github.io/supplementary-style-guide/#_product_names_and_version_references
- "Operator" must always be capitalized, see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization
- Suggested slight edit to explain _why_ someone would remove the resources (they're deprecated or unused). The abstract / introduction sentence should explain why someone would do a procedure.